server.go

```diff
@@ -12,26 +13,32 @@ import (
 
 func main() {
 	e := echo.New()
-	e.GET("/", func(c echo.Context) error {
-
-		data := getData()
-		return c.String(http.StatusOK, data)
-	})
+	e.GET("/", defaultRoute)
 	e.Start(":1111")
 }
 
-func getData() string {
-	response, err := http.Get("http://pokeapi.co/api/v2/pokedex/kanto/")
+func defaultRoute(c echo.Context) error {
+	data := getData()
+	return c.JSON(http.StatusOK, data)
+}
+
+func getData() map[string]interface{} {
```

Comment:
I change the return code into map, since it's a JSON anyway. This is closest to `Map<String, Object>` in Java.